### ISSUE TITLE
Please release latest version

### ISSUE THREAD

**Issue by AndreLouisCaron** [atomita/fluent-plugin-aws-elasticsearch-service#66]
#### Problem

I want to `gem install fluent-plugin-aws-elasticsearch-service` and get the latest version because it supports `fluent-plugin-elasticsearch>=4`, but this version is not yet available on Ruby gems.

#### Steps to replicate

```
$ gem install fluent-plugin-aws-elasticsearch-service:2.3.0
$ gem list | grep -i elastic
...
fluent-plugin-aws-elasticsearch-service (2.3.0)
fluent-plugin-elasticsearch (3.8.0)
```

#### Expected Behavior or What you need to ask

Please release the latest version on Ruby gems :-)

#### Using Fluentd and ES plugin versions

I am using Docker image `fluent/fluentd:v1.11.0-1.0` and running `gem install fluent-plugin-aws-elasticsearch-service`.

OS version:
```
$ uname -a
Linux 3dacadfd4385 4.19.76-linuxkit #1 SMP Tue May 26 11:42:35 UTC 2020 x86_64 Linux
```

FluentD version:
```
$ fluentd --version
fluentd 1.11.0
```

**Comment by cosmo0920** [comment]
@atomita Could you kindly take a look? Thanks in advance.

**Comment by atomita** [comment]
@AndreLouisCaron 
I released v2.4.0
Thank you.

@cosmo0920 
Thank you for your support.

**Comment by AndreLouisCaron** [comment]
Thanks a lot for releasing, it now works like a charm :-)
